docs/inference.rst

```diff
@@ -194,7 +195,7 @@ An example of generating an injection::
     COA_PHASE=1.5
     POLARIZATION=1.75
     DISTANCE=100000 # in kpc
-    INJ_F_MIN=28.
+    INJ_F_MIN=18.
```

Comment:
See answer above. Because I can't specify a f_ref using inspinj, this does mean the waveform wouldn't be exactly the same if the injection were precessing. However, since the example is for an aligned-spin waveform, that doesn't matter. Eventually we should just replace this with a call to `pycbc_create_injection` anyway.